android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java

```diff
@@ -320,6 +320,41 @@ BraveWalletService getBraveWalletService() {
         return null;
     }
 
+    // Lint complains on trendPercentage.setText()
```

Comment:
What was the exact complaint?

Reply:
@samartnik I cannot find it in my draft. Just removed `SuppressLint` and run the build again - I don't see any lint errors. But the recent re-build is after rebase including the new Chromium 96. Will push the fix with removing these lines.

Reply:
I have force-pushed the fix and removed the comment and suppress lint annotation